BICCN-policies/dataset-metadata.yaml

```diff
@@ -0,0 +1,85 @@
+identifier: REQUIRED ## Post upload (or during dandi organize)
+name: REQUIRED
```

Comment:
Prefer to call this "title" to encourage people to use a formal title, but doesn't matter so long as people provide appropriate info.